components/inputs/input-checkbox.js

```diff
@@ -151,6 +157,9 @@ class InputCheckbox extends FocusMixin(SkeletonMixin(RtlMixin(LitElement))) {
 					bottom: 0.3rem;
 					top: 0.3rem;
 				}
+				.d2l-input-inline-help {
+					margin-left: ${cssSizes.inputBoxSize + cssSizes.checkboxMargin}rem;
```

Comment:
We can change this to `margin-inline-start` to make this work for both LTR and RTL.

Our alternative (old) approach is:
```css
.d2l-input-inline-help {
    margin-left: ${cssSizes.inputBoxSize + cssSizes.checkboxMargin}rem;
}
:host([dir="rtl"]) .d2l-input-inline-help {
    margin-left: 0;
    margin-right: ${cssSizes.inputBoxSize + cssSizes.checkboxMargin}rem;
}
```